### PR TITLE
Yeezick/native applicants screen

### PR DESCRIPTION
**Pull request by yeezick**
This PR will create the applicants' screen. This version of the screen will pick the first project in `allProjects` by default until `TeamView` component is able to pass in projectID. 
This PR satisfies the completion of 
[BC-100: Applicants Screen](https://bootcamper.atlassian.net/jira/software/c/projects/BC/boards/2?modal=detail&selectedIssue=BC-100) 
excluding the addition of "personal message". This screen should be revisited once messages are added to the schema.

### CODE REVIEW

File: native/components/SideMenu.jsx [hunk 1]
```diff
@@ -52,48 +53,18 @@ export const SideMenu = () => {
...
+        <Drawer.Screen name="CreateProject" component={CreateProject} />
         <Drawer.Screen name="EditProfile" component={EditProfile} />
+        <Drawer.Screen name="EditProject" component={EditProject} />
+        <Drawer.Screen name="Roulette" component={Roulette} />
+        <Drawer.Screen name="SingleProject" component={SingleProject} />
+        <Drawer.Screen name="SignIn" component={SignIn} />
+        <Drawer.Screen name="SignUp" component={SignUp} />
+        <Drawer.Screen name="UserDashboard" component={UserDashboard} />
+        <Drawer.Screen name="UserProfile" component={UserProfile} />
       </Drawer.Navigator>
```
Comment (gooliajulia) [inline native/components/SideMenu.jsx:68]: I'm liking this layout change. looks really clean!

File: native/screens/Applicants/Applicants.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    setEngineers(() => filterRoles('Software Engineer'));
+    setDesigners(() => filterRoles('UX Designer'));
+  }, [reduxUserID, soloProject]);
+
+  if (verifiedOwner) {
+    return (
+      <ScrollView>
+        <Text>Applications</Text>
+        <RoleList applicants={engineers} navigation={navigation} role={'Software Developers'} />
+        <Text>{'\nJUST A DIVIDER \n DONT MIND ME \n DELETE ME AFTER \n'}</Text>
```
Comment (gooliajulia) [inline native/screens/Applicants/Applicants.jsx:35]: nothing to see here

File: native/screens/Applicants/Applicants.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  useEffect(() => {
+    if (!applicants) {
+      return;
+    } else if (loadMore) {
+      setVisibleList(applicants);
+    } else {
+      setVisibleList(applicants.slice(0, 3));
+    }
+  }, [loadMore, applicants]);
+
```
Comment (gooliajulia) [inline native/screens/Applicants/Applicants.jsx:57]: I like this approach for the 'load more' functionality. Can't wait to see it in action!

**Review by gooliajulia — APPROVED** [review]
Way to turn a 2 pointer into a 1 pointer 🚀  Can't wait to see it in action.

File: native/screens/Roulette.jsx [hunk 4]
```diff
@@ -56,7 +57,7 @@ const ProjectInfo = ({ project }) => {
       <View className="roulette-tools">
         <Text> Built with:</Text>
         {project.tools?.map((tool) => (
-          <Text>{tool}</Text>
+          <Text key={uuid.v4()}>{tool.name}</Text>
         ))}
       </View>
       <Text>{`Looking for collaborators who can commit at least X hours per week`}</Text>
```
Comment (flexibleidealist) [inline native/screens/Roulette.jsx:63]: The number of hours could be added dynamically based on the `time_commitment` property; I used a switch statement in the SingleProject screen to add the number of hours that should work here, too.

**Review by flexibleidealist — APPROVED** [review]
I think this looks good! Eventually CreateProject and EditProject are going to be the same screen component with different behavior based on the props it receives when it's called so these won't be two separate screens for these.

File: native/screens/Roulette.jsx (hunk 4 above)
Comment (yeezick) [reply native/screens/Roulette.jsx:63]: True! I'll add it to Jira

**Pull request merged**